### PR TITLE
fix: remove unneeded uses of ECDSA private key

### PR DESCRIPTION
**Pull request by Oppen**
Both the operator and aggregator were using the `BuildAll` helper that required a private key due to creation of signers, only to discard the signers and keep the reader and subscriber.
Changing it to `BuildReadClients` allows removing some of it, in the case of the operator meaning only registration requires the ECDSA key to be present, reducing risks of losing funds.

A later change should update the docs to note the ECDSA key is unnecessary after registration and should be deleted from the operator. Left for later as a bigger restructure of the operator guide is needed.

## Type of change

- [x] Refactor

## Checklist

- [x] This change requires new documentation.
  - [ ] Documentation has been added/updated.
- [x] Known issue: this breaks operator metrics until they start signing their announcement with BLS. @MarcosNicolau is working on that on another PR.

### CODE REVIEW

**Review by MarcosNicolau — COMMENTED** [review]
Everything works fine! We need to update the documentation regarding the management of the `ecdsa` key.

**Review by JulianVentura — COMMENTED** [review]
Code looks fine and it seems to be working. I'll test again when the telemetry fix is merged

**Review by MarcosNicolau — APPROVED** [review]

**Review by JuArce — APPROVED** [review]
Approved, but once #1488 it will have merge conflicts

File: core/config/operator.go
Comment (uri-99) [inline core/config/operator.go:1]: line 48 should also be removed

**Review by uri-99 — CHANGES_REQUESTED** [review]

**Review by uri-99 — APPROVED** [review]

**Pull request merged**
